src/generator/models/javascript.ts

```diff
@@ -99,6 +104,29 @@ export function prismaFieldToNexusType(field: DMMF.Field) {
   }
 }
 
+export function prismaFieldToNexusResolver(model: DMMF.Model, field: DMMF.Field) {
+  if (field.kind === 'object') {
+    return (root: any, _args: any, ctx: any) => {
```

Comment:
Probably `unknown` is the right type to use here.

Reply:
with `unknown` i cannot acces to `ctx.prisma` or pass `root` to constraint functions

Reply:
Yeah, where you find `any` working best go for it. With `unknown` you'll need to "test" it before use. This could be a generated validation step in the resolvers. Later we can have that validation disabled for production (even stripped out entirely at build time) to optimize production performance.

But `any` is fine for now.